.github/workflows/aws-moped-sqs-helper.sh

```diff
@@ -178,3 +178,59 @@ function deploy_event_function {
   cd $MAIN_DIR;
   echo "Exit, current path: ${PWD}";
 }
+
+#
+# Builds & Deploys Event Function for Moped Test Instances
```

Comment:
I went too far down the road of recycling this bash script, and I ended up having to add functions and make some modifications. I could have used boto3 here, but I'm not sure what would have been faster. 🤷

Reply:
I'd say whatever works is a great solution. I think of a prefect task having a single job to do, and as long as it works, no worries on what gets it done. 👍 Tasks are like the API of getting-stuff-done.